simple_use_case/source/postprocessing.py

```diff
@@ -13,40 +12,42 @@
 
 
 def main(args):
-    pvd_file = args.pvd
+    pvd_file = args["inputfile"]
     source = PVDReader(registrationName="poisson.pvd", FileName=[pvd_file])
-    source.PointArrays = ["u"]
+    source.PointArrays = [args["field"]]
+    UpdatePipeline()
 
+    (xmin, xmax, ymin, ymax, zmin, zmax) = source.GetDataInformation().GetBounds()
+    # init the 'Line' selected for 'Source'
     plotOverLine1 = PlotOverLine(
         registrationName="PlotOverLine1", Input=source, Source="Line"
     )
-    # init the 'Line' selected for 'Source'
-    plotOverLine1.Source.Point1 = [0.0, 0.0, 0.0]
-    plotOverLine1.Source.Point2 = [1.0, 1.0, 0.0]
+    plotOverLine1.Source.Point1 = [xmin, ymin, zmin]
+    plotOverLine1.Source.Point2 = [xmax, ymax, zmax]
     UpdatePipeline()
 
     # save data
     SaveData(
-        args.csv,
+        args["outputfile"],
         proxy=plotOverLine1,
         ChooseArraysToWrite=1,
-        PointDataArrays=["arc_length", "u", "vtkValidPointMask"],
+        PointDataArrays=["arc_length", args["field"], "vtkValidPointMask"],
     )
 
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(
         prog=f"pvbatch {__file__}",
         description="Plots the solution over a line and writes the data to file.",
-        usage="%(prog)s [options] pvd csv",
+        usage="%(prog)s [options] ",
     )
-    parser.add_argument("pvd", type=str, help="The source pvd filepath.")
-    parser.add_argument("csv", type=str, help="The target csv filepath.")
+    parser.add_argument("-i", "--inputfile", required=True, help="The source pvd filepath.")
+    parser.add_argument("-o", "--outputfile", required=True, help="The target csv filepath.")
     parser.add_argument(
         "--field",
         type=str,
         default="u",
         help="Field variable to plot (default: u)",
     )
-    args = parser.parse_args(sys.argv[1:])
+    args = vars(parser.parse_args())
```

Comment:
Is the change to this file related to this PR? It seems to me to just be some cleanup related to runtime arg handling, no? If so, I would do this in a separate PR, because this changes the script usage and thus we have to port all implementations. I guess it's easier to review if that's a separate PR. But I may be mistaking and it is related to the issue adressed in this PR

Reply:
The reason why I made changes to this file is that I had to allow for the arbitrary domain size. Before the line plot was hardcoded to the unit square. IMO this particular commit should be included in #32 which is what this PR is proposing.

Reply:
yes I saw that you now get the bounds from the vtk file, that's nice. And I agree that that may be better placed in #32, although I have no strong opinion about that because we merge this one into #32, anyway...

But I was talking about your changes to the script arguments. You changed `pvd` and `csv` to `-i` and `-o`, no? I like that, but I think it is a different concern and may be easier to change in a separate PR because we have to adjust the calls to this script in all WF realizations, and these changes would clutter the diff to review here and are also unrelated to the task of this PR. Or maybe I am missing something.. What do you think?

Reply:
okay, I think you are right. The change for that specific line with the script arguments is a different concern. I just thought it would not make a big difference since we have to adjust all the other workflow implementations to also use domain size and numdofs. 
I have no strong opinion about in which PR this change should be made, so I will just follow your suggestion and revert the change in the CLI for this PR.